Makefile

```diff
@@ -187,7 +187,12 @@ publish_mix: compile
 
 #==> Tests tasks
 
-test: test_erlang test_elixir
+test:
+	$(RM) -r .home
+	mkdir .home
+	HOME=$(CURDIR)/.home $(MAKE) test_erlang
```

Comment:
So I just ran the test suite and I have verified that it does not create anything on HOME. So I would keep things as is for now. :) It could even help us detect accidentally writing to home in the future, which is a bad practice in a test suite anyway.